### PR TITLE
Add "--name-only" to "git diff" autocomplete

### PR DESCRIPTION
**Pull request by atsalolikhin-spokeo**
The main change is:  resolves email to `hello@withfig.com` with the subject `Fig does not know about "git diff --name-only"`

Incidental changes: 
- Remove `isOptional: false` to stop the linter complaining that we are explicitly setting `isOptional` when the default for it is `false` anyway
- Run `npm run lint:fix`

### CODE REVIEW

**Comment by atsalolikhin-spokeo** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by atsaloli** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by atsaloli** [comment]
recheck

**Comment by brendanfalk** [comment]
@grant0417 what's going on with these lints?

**Comment by atsalolikhin-spokeo** [comment]
I didn't change `svn.ts`.

**Comment by atsalolikhin-spokeo** [comment]
I added commit 0777b98482b344d4121c5dfdcbea7f751c00cfce to fix the thing the yarn linter is complaining about.

**Comment by grant0417** [comment]
It seems like it was an issue with a pr that didnt pass the checks was merged, @atsalolikhin-spokeo it seems like the git spec is not passing lints, can you run `npm run lint:fix` and commit that back?

**Comment by atsalolikhin-spokeo** [comment]
> It seems like it was an issue with a pr that didnt pass the checks was merged, @atsalolikhin-spokeo it seems like the git spec is not passing lints, can you run `npm run lint:fix` and commit that back?

Done in commit 8e6e6937

**Comment by atsalolikhin-spokeo** [comment]
@grant0417 How are we looking now?

**Comment by grant0417** [comment]
Seems you modified the `yarn.lock` are you able to revert that?

**Comment by atsalolikhin-spokeo** [comment]
@grant0417 I've reverted the unintended change to `yarn.lock`.  Thanks!

File: package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "cors": "^2.8.5",
     "danger": "^11.3.0",
     "esbuild": "^0.19.4",
-    "eslint": "^8.50.0",
+    "eslint": "^8.51.0",
```
Comment (grant0417) [inline package.json:83]:
```suggestion
    "eslint": "^8.50.0",
```

Comment (grant0417) [reply package.json:83]: Last thing for merge!

Comment (atsalolikhin-spokeo) [reply package.json]: Fixed in  commit b70cfe59a0041614bdc63836aab5f767f35e3c78  Thanks!

**Comment by grant0417** [comment]
Thanks!

**Pull request merged**